packages/calendar/src/components/ui/date-time-input/DateTextInput.tsx

```diff
@@ -65,9 +68,9 @@ export const DateTextInput: React.FC<DateTextInputProps<{}>> = ({
   const { colors } = useThemeFields(
     {
       colors: {
-        backgroundColor: dateInputTheme.backgroundColor,
-        borderColor: dateInputTheme.borderColor,
-        backgroundColorInvalidDate: dateInputTheme.backgroundColorInvalidDate
+        backgroundColor: textInputTheme.backgroundColor,
+        borderColor: textInputTheme.borderColor,
+        backgroundColorInvalidDate: textInputTheme.backgroundColorInvalidDate
```

Comment:
Maybe add `backgroundColorValidationError` to `StandardTextInputTheme`, and an `invalidInput` prop to `StandardTextInput`?

This way we get a standardized look for validation error highlight, and `DateTextInput` can just set the `invalidInput` prop when invalid date.